CONTRIBUTING.md

```diff
@@ -153,8 +153,7 @@ $ script/cibuild     # runs everything, with verbose debug output
 ## Updating 3rd party packages
 
 1. Update `go.mod`.
```

Comment:
It occurs to me on a second look that we actually should mention something here about updating `go.sum` as well, since it is tracked in Git.  Perhaps something about running `go mod tidy` to update `go.sum` would suffice, or possibly `make go.sum` (which [relies](https://github.com/git-lfs/git-lfs/blob/2137b4976f4c6e4af3c17bcedb87a1f23270caf2/Makefile#L585-L587) on `go mod verify` at the moment)?

(I also notice we have [some](https://github.com/git-lfs/git-lfs/blob/2137b4976f4c6e4af3c17bcedb87a1f23270caf2/Makefile#L105-L107) [stale](https://github.com/git-lfs/git-lfs/blob/2137b4976f4c6e4af3c17bcedb87a1f23270caf2/Makefile#L268) [references](https://github.com/git-lfs/git-lfs/blob/2137b4976f4c6e4af3c17bcedb87a1f23270caf2/Makefile#L589-L621) to the `vendor` directory in the `Makefile`; those are likely best dealt with in another PR, though.)

Would you be willing to add some text here regarding `go.sum`, before we merge this PR?

Thanks again for the help!

Reply:
Done.

Reply:
Fantastic, thanks very much!